app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -134,15 +132,14 @@ class ProfileEditActivityTest {
   }
 
   @Test
-  // TODO(#973): Fix ProfileEditActivityTest
-  @Ignore
-  fun testProfileEditActivity_startActivityWithAdminProfile_checkAdminInfoIsDisplayed() {
+  fun testProfileEdit_startWithAdminProfile_checkAdminInfoIsDisplayed() {
     ActivityScenario.launch<ProfileEditActivity>(
```

Comment:
One last nit change, else it's good to go. 
Rather than import `ActivityScenario` as a whole, we only need `launch`.

```diff
- import androidx.test.core.app.ActivityScenario
+ import androidx.test.core.app.ActivityScenario.launch
```

Reply:
DONE